libs/src/sdk/services/DiscoveryNodeSelector/healthChecks.ts

```diff
@@ -147,7 +147,7 @@ export const parseHealthStatusReason = ({
   if (minVersion) {
     if (
       !data.version ||
-      !hasSameMajorAndMinorVersion(data.version, minVersion)
+      !hasGteMajorAndMinorVersions(data.version, minVersion)
```

Comment:
I think the check right after this handles this, so we don't need this check if we decide we don't want to stick to the same major/minor pair